code/__DEFINES/economy_defines.dm

```diff
@@ -60,8 +60,25 @@
 #define SUPPLY_MATERIALS 	7
 #define SUPPLY_MISC 		8
 #define SUPPLY_VEND 		9
+#define SUPPLY_SHUTTLE		10
 
 #define DEFAULT_CRATE_VALUE 15
 
 /// mail deliveries
 #define MAIL_DELIVERY_BONUS	100
+
+/// Shuttle types that can be ordered, along with prices
+// TODO add prices
+GLOBAL_LIST_INIT(orderable_shuttle_types, list(
+	/datum/map_template/shuttle/emergency/bar,
+	/datum/map_template/shuttle/emergency/dept,
+	/datum/map_template/shuttle/emergency/military,
+	/datum/map_template/shuttle/emergency/clown,
+	/datum/map_template/shuttle/emergency/cramped,
+	/datum/map_template/shuttle/emergency/narnar,
+	/datum/map_template/shuttle/emergency/old,
+	/datum/map_template/shuttle/emergency/jungle,
+	/datum/map_template/shuttle/admin/skipjack  // hehe
+
+
+))
```

Comment:
just put this on SSshutles or SSeconomy tbh

Reply:
I don't think this even gets used anymore tbh, all the orderable shuttles are now datumized